pkg/charts/linkerd2/values_test.go

```diff
@@ -34,6 +34,14 @@ func TestNewValues(t *testing.T) {
 	namespaceSelectorSimple := &metav1.LabelSelector{MatchExpressions: matchExpressionsSimple}
 	namespaceSelectorInjector := &metav1.LabelSelector{MatchExpressions: matchExpressionsInjector}
 
+	defaultUnavailable := "25%"
+	defaultSurge := "25%"
+	defaultDeploymentStrategy := &map[string]interface{}{
+		"RollingUpdate": &map[string]interface{}{
+			"MaxUnavailable": &defaultUnavailable,
+			"MaxSurge":       &defaultSurge,
+		},
+	}
```

Comment:
I believe all of these keys should be `lowerCamelCase`. Also should `maxUnavailable` be `1`?